### PR TITLE
NFS-398: adds command to call the share function

### PR DESCRIPTION
**Pull request by Ninerian**
## Description

Adds the sharing command

## Related Issue

NFS-398

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** part in the readme.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Test plan:

- code review

### CODE REVIEW

File: src/lib/connection/connector/fallback-handlers.js [hunk 1]
```diff
@@ -96,13 +96,13 @@ export const openLink = url => {
 };
 
 /**
- * Open an external link
+ * Handler for unpossible functions
  *
- * @param {String} url address
+ * @param {String} cmd command name
  */
-export const nativeUpload = url => {
```
Comment (FenziRocks) [inline src/lib/connection/connector/fallback-handlers.js]: Why have you changed nativeUpload?
That should be not gone.

Comment (Ninerian) [reply src/lib/connection/connector/fallback-handlers.js]: Look downward. I did a unsupported function and merged the two cases. After looking through the code, I remembered, that we decided to end always in the fallback and throw warnings for features which are not doable with fallbacks.

**Comment by FenziRocks** [comment]
We wanted to catch, that we are native in our SDK, shouldn't we?

**Comment by FenziRocks** [comment]
@Ninerian Travis is pending.
I will adjust the usage.md

File: src/lib/connection/connector/fallback.js [hunk 2]
```diff
@@ -62,14 +62,15 @@ export const sendMessage = async (cmd, ...payload) => {
...
       return fallbacks.openLink.apply(null, payload);
-    case action.nativeUpload:
-      return fallbacks.nativeUpload();
     case action.langInfos:
       return fallbacks.langInfos();
     case action.branchDefaultLang:
       return fallbacks.getBranchDefaultLanguage();
     case action.prefContentLang:
       return fallbacks.getPreferredContentLocale.apply(null, payload);
+    case action.nativeUpload:
```
Comment (m-seidel) [inline src/lib/connection/connector/fallback.js]: Why is this case statement empty?

**Comment by m-seidel** [comment]
@Ninerian I fixed the empty switch clause in `src/lib/connection/connector/fallback.js` too

**Comment by FenziRocks** [comment]
Sorry but I checked yarn audit and it says something bad :(
image

File: package.json [hunk 3]
```diff
@@ -40,7 +40,6 @@
   },
   "homepage": "https://github.com/Staffbase/plugins-client-sdk#readme",
   "release": {
-    "branch": "master",
```
Comment (FenziRocks) [inline package.json]: Is this right. I just ask, if you did it for purpose.

Comment (Ninerian) [reply package.json]: Yes in the new version, the standard is good enough. It defaults to `['+([0-9])?(.{+([0-9]),x}).x', 'master', 'next', 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}]` which gives more possibilities.

**Pull request merged**